test/integration/targets/iosxr_lacp_interfaces/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- debug:
+      msg: "START iosxr_lacp_interfaces overridden integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- block:
+    - name: Overridde all interface LACP configuration with provided configuration
+      iosxr_lacp_interfaces: &overridden
+        config:
+          - name: Bundle-Ether12
+            churn_logging: both
+            collector_max_delay: 100
+            switchover_suppress_flaps: 500
+
+          - name: GigabitEthernet0/0/0/1
+            period: 300
+        state: overridden
+      register: result
+
+    - name: Assert that correct set of commands were generated
+      assert:
+        that:
+          - "{{ overridden['commands'] | symmetric_difference(result['commands']) |length == 0 }}"
+
+    - name: Assert that before dicts are correctly generated
+      assert:
+        that:
+          - "{{ populate | symmetric_difference(result['before']) |length == 0 }}"
+
+    - name: Assert that after dict is correctly generated
+      assert:
+        that:
+          - "{{ overridden['after'] | symmetric_difference(result['after']) |length == 0 }}"
+
+    - name: Overridde all interface LACP configuration with provided configuration (IDEMPOTENT)
+      iosxr_lacp_interfaces: *overridden
+      register: result
+
+    - name: Assert that task was idempotent
+      assert:
+       that:
+         - "result['changed'] == false"
```

Comment:
Same like above.

Reply:
If result.changed would not be false if commands list is not empty.